CMakeLists.txt

```diff
@@ -79,6 +79,16 @@ set(CMAKE_VERBOSE_MAKEFILE
     false
     CACHE STRING "Show all outputs including compiler lines." FORCE)
 
+# Force colored output (mainly for Ninja)
+option (FORCE_COLORED_OUTPUT "Always produce ANSI-colored output." FALSE)
```

Comment:
I would prefer if we set the default to true if Ninja is used as generator. Then you won't need the change in `configure`.

Reply:
Addressed this change.